frontend/src/main/sign-out/SignOutButton.js

```diff
@@ -0,0 +1,20 @@
+import { Button } from '@material-ui/core'
+import { useAuth } from 'context/Auth'
+import React from 'react'
+import authApi from 'api/AuthApi'
+
+const SignOutButton = (props) => {
+    const [auth, setAuth] = useAuth()
+
+    const handleClick = async () => {
+        // @ts-ignore
```

Comment:
usually this is not needed

Reply:
This is a way to suppress a warning from the linter
![image](https://user-images.githubusercontent.com/6403398/119580025-90be7d80-bdb7-11eb-9512-ea67070d4b8f.png)